app-text/sigil/sigil-1.0.0.ebuild

```diff
@@ -0,0 +1,83 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6,7} )
```

Comment:
py3.5 support will be dropped soon, you can still leave it, but you should start testing py3.8 support on these contributions if possible.

Reply:
I prefer that py3.5 support is removed with all other python packages.
For py3.8 support I need `dev-python/cssutils` and `dev-python/css-parser` to support py3.8.